packages/client/src/pages/Destination/NearbyDestCard/NearbyDestCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './nearbyDestCard.scss';
+import { Image } from '../../../components/Image/Image';
+import { isEmpty } from 'lodash';
+import Destination from '../../../../../shared/models/destination';
+import { RatingBar } from '../../../components/Rating/RatingBar';
+
+interface NearbyDestCardProps {
+    index: number;
+    destination: Destination;
+}
+
+export class NearbyDestCard extends React.Component<NearbyDestCardProps> {
+    render() {
+        return (
+            <div className="nearby-dest-wrapper">
+                <div className="nearby-dest-image">
+                    <Image
+                        src={
+                            isEmpty(this.props.destination.photoRefs)
+                                ? null
+                                : this.props.destination.photoRefs[0]
```

Comment:
nani but why is this working?? I thought we need to send it to the server to retrieve the photo lol

Reply:
> > I realized that the map doesn't really tell a lot without those numberings... so I refactored the StaticMap class a little to get Marker objects directly as props, and build that in to request params. One implication for this change is that because each marker has unique number label, we can no longer group them under the same color in our request. Not sure if this is significant but let me know if it is LOL @knox153
> 
> Adding numbering was a good idea! 👍
> It makes the url longer, and there is a 8192 characters limit in API urls. But I doubt we will reach that so it should be fine haha

Ohhh i see hahaha. If someone could make a trecipe that takes > 8192 characters for map request, I will gladly refactor it for that person. But until then.... 👋

Reply:
> nani but why is this working?? I thought we need to send it to the server to retrieve the photo lol

LOL since for the whole destination page, we r calling place details request regardless of whether the destination was stored/not.
```
            photoRefs: place.photos
                ? place.photos.map((photo) => photo.getUrl({ maxHeight: 100 }))
                : [],
```
This, in `getDestModel` of `DestinationPage` is the secret.
I better document this I guess hahaha

Reply:
Ohhhhh, so we are not completely abandoning client side fetching haha